webapp/channels/src/packages/mattermost-redux/src/action_types/files.ts

```diff
@@ -13,4 +13,6 @@ export default keyMirror({
     RECEIVED_FILES_FOR_POST: null,
     RECEIVED_UPLOAD_FILES: null,
     RECEIVED_FILE_PUBLIC_LINK: null,
+
+    REMOVED_FILE: null,
```

Comment:
a bit more descriptive? REMOVED_FILE_ from store or somethinge

Reply:
These are redux types, they are implicitly all intended for store, like all constants in this file.